javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2401,6 +2449,152 @@ public static List<? extends ExpressionTree> caseTreeGetExpressions(CaseTree cas
     }
   }
 
+  /**
+   * Get the list of expressions from a case expression. For the default case, this is empty.
```

Comment:
"expressions" in "list of expressions" is ambiguous.  Please clarify.
"from a case expression": I'm not sure what that is.  Is this a case constant, a case pattern, a guard expression, or something else?  I think it might be case constants.